api/openapi.yaml

```diff
@@ -1652,6 +1652,8 @@ components:
               type: boolean
             resourceCleanup:
               $ref: '#/components/schemas/ResourceCleanupState'
+            stats:
+              $ref: '#/components/schemas/AssetScanStats'
```

Comment:
I think we also need to add a startTime and endTime to the AssetScan itself. The start time should be set from the the moment the AssetScan moves from Pending to Scheduled and scanEndTime should be set when the AssetScan moves to Done or Failed. That will allow us to calculate the duration the AssetScan really took to scan from the point we start creating resources. We can also add cleanupEndTime, which is the point when the cleanup finishes which will give us the full e2e time for an AssetScan.

Adding these fields is something we need anyway in order to implement a per AssetScan timeout and I'm sure there are other use cases too.

Reply:
Not sure this is part of this PR though. This is not part of the scan benchmarking, but times that took to the orchestrator to run the scan

Reply:
Added issue https://github.com/openclarity/vmclarity/issues/509